aten/src/ATen/cudnn/Types.cpp

```diff
@@ -29,6 +29,13 @@ cudnnDataType_t getCudnnDataTypeFromScalarType(const at::ScalarType dtype) {
 }
 
 cudnnDataType_t getCudnnDataType(const at::Tensor& tensor) {
+  if (tensor.is_quantized()) {
```

Comment:
should we add this to `getCudnnDataTypeFromScalarType`?

Reply:
It seems we never call `getCudnnDataTypeFromScalarType` directly since it's called from `getCudnnDataType` so I was thinking it'd be better to do it from the calling function but maybe it's clearer to do it your way. i can make that change